assembly/enduser-files.xml

```diff
@@ -0,0 +1,28 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<assembly xmlns="http://maven.apache.org/ASSEMBLY/2.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/ASSEMBLY/2.0.0 http://maven.apache.org/xsd/assembly-2.0.0.xsd">
+
+    <id>enduser-files</id>
+
+    <formats>
+        <format>zip</format>
```

Comment:
I'd also add a tar.gz format, for unix users...
```
  <format>tar.gz</format>
```

Reply:
Good point. I added `tar.gz` in 3b7ad2d644121f9671c5441cb62eaa3a663bb7a6. We can see that both `tar.gz` and `zip` files are [generated](https://xspec.visualstudio.com/xspec/_build/results?buildId=1367&view=logs&j=166b8c02-95cc-5502-dd2b-44f9c8527e72&t=b82422a0-28d8-50bc-bee6-b0195d090944&l=49) by ` mvn package -P release`.

As for the `zip` file, I expect it to be also reused as the [Oxygen add-on package](https://github.com/xspec/xspec/wiki/Running-with-Oxygen#loading-xspec-via-oxygen-add-on). Right now, the add-on package is the whole source code archive containing many files useless at run time.